cli/src/pcluster/validators/cluster_validators.py

```diff
@@ -136,6 +137,22 @@ def _validate(self, os, scheduler):
             )
 
 
+class PrivateOsCustomAmiValidator(Validator):
+    """If OS is one of the OSSes that we do not publicly publish/release we need to use a Custom AMI."""
+
+    def _validate(self, os: str, custom_ami: str):
+        if not custom_ami and os in PRIVATE_OSSES:
+            self._add_failure(
+                (
+                    # TODO: Add Link
+                    f"Parallelcluster doesnt publish/release official {os} AMI. "
```

Comment:
`ParallelCluster` is the correct name of the product.

I'd write: `There is no official AMI for {os}.  Please build your own AMI using `pcluster build-image` command, as explained in the documentation: https://docs.aws.amazon.com/parallelcluster/latest/ug/building-custom-ami-v3.html`